src/clarity-angular/buttons/_buttons.clarity.scss

```diff
@@ -215,6 +241,64 @@ $clr-button-appearance-map: (
         @include generateButton();
         @include clr-getStylesFromMap($clr-button-appearance-map, form);
         @include populateButtonProperties(default);
+    }
+
+    .btn-group {
+        //.btn-outline-<state> is the new BS4 naming convention
+        //TODO: we support both the previous one and the new one but we will deprecate the old ones from the example in 0.5.0
+        &.btn-secondary > .btn,
```

Comment:
My main issue is that we are applying button classes to button groups which is requiring us to introduce unnecessary specificity and bloat our CSS as shown below.

`.btn-group--primary` is the direction we want to go, right? I see little difference between choosing `.btn-group-primary` or `.btn-group--primary` beyond another breaking change when we finalize our CSS naming conventions in code.

What I am against is `.btn-group.btn-whatever > .btn`. I think that's an unnecessary layer of specificity as compared to `.btn-group-whatever > .btn`.


Given how eager we have been to rip off the band aid with other components, this level of reluctance seems out of character.